src/renderer/missionWindow/MissionWindow.tsx

```diff
@@ -223,50 +227,98 @@ export default class MissionWindow extends Component<ThemeProps, State> {
     const name = event.target.name as GeofenceChecklistType;
     const value = parseInt(event.target.value, 10) || 0;
     switch (name) {
-      case 'geofenceTop':
+      case 'geofenceKeepInTop':
         ipc.postUpdateBoundingBoxes(true, {
-          name: 'Geofencing',
+          name: 'geofenceContainerKeepIn',
           bounds: {
             top: value,
-            bottom: checklist.geofenceBottom as number,
-            left: checklist.geofenceLeft as number,
-            right: checklist.geofenceRight as number,
+            bottom: checklist.geofenceKeepInBottom as number,
+            left: checklist.geofenceKeepInLeft as number,
+            right: checklist.geofenceKeepInRight as number,
           },
         });
         break;
 
-      case 'geofenceLeft':
+      case 'geofenceKeepInLeft':
         ipc.postUpdateBoundingBoxes(true, {
-          name: 'Geofencing',
+          name: 'geofenceContainerKeepIn',
           bounds: {
-            top: checklist.geofenceTop as number,
-            bottom: checklist.geofenceBottom as number,
+            top: checklist.geofenceKeepInTop as number,
+            bottom: checklist.geofenceKeepInBottom as number,
             left: value,
-            right: checklist.geofenceRight as number,
+            right: checklist.geofenceKeepInRight as number,
           },
         });
         break;
 
-      case 'geofenceRight':
+      case 'geofenceKeepInRight':
         ipc.postUpdateBoundingBoxes(true, {
-          name: 'Geofencing',
+          name: 'geofenceContainerKeepIn',
           bounds: {
-            top: checklist.geofenceTop as number,
-            bottom: checklist.geofenceBottom as number,
-            left: checklist.geofenceLeft as number,
+            top: checklist.geofenceKeepInTop as number,
+            bottom: checklist.geofenceKeepInBottom as number,
+            left: checklist.geofenceKeepInLeft as number,
             right: value,
           },
         });
         break;
 
-      case 'geofenceBottom':
+      case 'geofenceKeepInBottom':
         ipc.postUpdateBoundingBoxes(true, {
-          name: 'Geofencing',
+          name: 'geofenceContainerKeepIn',
           bounds: {
-            top: checklist.geofenceTop as number,
+            top: checklist.geofenceKeepInTop as number,
             bottom: value,
-            left: checklist.geofenceLeft as number,
-            right: checklist.geofenceRight as number,
+            left: checklist.geofenceKeepInLeft as number,
+            right: checklist.geofenceKeepInRight as number,
+          },
+        });
+        break;
+
+      case 'geofenceKeepOutTop':
+        ipc.postUpdateBoundingBoxes(true, {
+          name: 'geofenceContainerKeepOut',
```

Comment:
call it Geofence Keep Out or Geofence Keep-Out cause in the future hovering over the geofence box will pop up this string